test/integration/admin/api/registry/policies_controller_test.rb

```diff
@@ -21,14 +21,44 @@ def setup
 
   test 'POST create responds with an error message when it is incorrect' do
     FactoryBot.create(:policy, policy_params[:policy].merge(account: @provider))
-    assert_no_difference(@provider.policies.method(:count)) do
+    assert_no_difference(Policy.method(:count)) do
       post admin_api_registry_policies_path(policy_params)
     end
     assert_response :unprocessable_entity
     assert_equal ['has already been taken'], JSON.parse(response.body).dig('errors', 'version')
   end
 
-  def policy_params
-    { policy: {name: 'my-name', version: 'my-version', schema: '{"foo": "bar"}'}, access_token: @access_token.value }
+  test 'POST create verifies permission' do
```

Comment:
_(This is just a comment)_
Personally I like each assertion in a different test case, makes it a bit more readable when running them:
```
test 'POST token returns forbidden when wrong scope'
test 'POST token returns forbidden when wrong permissions'
test 'POST token returns created when correct scope'
```

Reply:
Yes normally there should be only one assertion per test but we are not following that rule. Otherwise our tests might take days to finish running